samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt

```diff
@@ -21,14 +25,42 @@ import org.mozilla.samples.browser.R
  */
 @Suppress("LargeClass")
 class InstalledAddonDetailsActivity : AppCompatActivity() {
+    private val scope = CoroutineScope(Dispatchers.IO)
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_installed_add_on_details)
         val addon = requireNotNull(intent.getParcelableExtra<Addon>("add_on"))
-        bind(addon)
+        bindAddon(addon)
+    }
+
+    private fun bindAddon(addon: Addon) {
+        scope.launch {
+            try {
+                val context = baseContext
+                val addons = context.components.addonManager.getAddons()
+                scope.launch(Dispatchers.Main) {
+                    addons.find { addon.id == it.id }.let {
+                        if (it == null) {
+                            throw AddonManagerException(NullPointerException())
```

Comment:
nit: we could use the same approach as https://github.com/mozilla-mobile/fenix/pull/10531#discussion_r422329976